app/helpers/application_helper/toolbar/dialog_center.rb

```diff
@@ -34,58 +34,4 @@ class ApplicationHelper::Toolbar::DialogCenter < ApplicationHelper::Toolbar::Bas
       ]
     ),
   ])
-  button_group('dialog_add', [
-    select(
-      :dialog_add_choice,
-      'pficon pficon-add-circle-o fa-lg',
-      N_('Add'),
-      nil,
-      :enabled => true,
-      :items   => [
-        button(
-          :dialog_add_tab,
-          'pficon pficon-add-circle-o fa-lg',
-          t = N_('Add a new Tab to this Dialog'),
-          t,
-          :url_parms => "?typ=tab&id=\#{@edit[:rec_id]}",
-          :klass     => ApplicationHelper::Button::DialogAddTab),
```

Comment:
toolbar buttons can be removed:

app/helpers/application_helper/button/dialog_add_tab.rb
spec/helpers/application_helper/buttons/dialog_add_tab_spec.rb